src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -48,5 +48,11 @@
     public static final String ACCEPT_SUB_FOLDER_NAME = "accept";
     public static final String REJECT_SUB_FOLDER_NAME = "reject";
     public static final String FORWARD_SLASH = "/";
-    public static final String CONCUR_PROCESSED_FILE_DATE_FORMAT = "yyyy-MM-dd-HHmmss";
+    public static final String CONCUR_DATE_FORMAT = "yyyy-MM-dd";
+    public static final String CONCUR_PROCESSED_FILE_DATE_FORMAT = CONCUR_DATE_FORMAT + "-dd-HHmmss";
```

Comment:
CONCUR_DATE_FORMAT has the "-dd" in it, but the extra suffix in CONCUR_PROCESSED_FILE_DATE_FORMAT also adds another "-dd" of its own. When the concatenation occurs above, the final value has "-dd-dd" instead of "-dd" in it.